RELEASE/scripts/autoscend/paths/auto_path_util.ash

```diff
@@ -3,7 +3,7 @@
 boolean auto_buySkills()  // This handles skill acquisition for general paths
 {
 	// TODO: Torso Awareness is worth obtaining in other cases too.
-	if((my_meat() >= 10000)
+	if((my_meat() >= get_property("auto_torsoAcquirePrice").to_int())
```

Comment:
```suggestion
	if((my_meat() >= meatReserve() + 6000)
```
use above if we skip on the settings entirely.

alternatively. rename the setting from `auto_torsoAcquirePrice` to `auto_torsoAwarenessMeatReserve` which is "additional meat to reserve in case of buying torso awareness. which should default to 1000 meat.
```suggestion
	if((my_meat() >= meatReserve() + 5000 + get_property("auto_torsoAwarenessMeatReserve").to_int())
```

Reply:
For the first suggestion, I'd choose that but I'm curious why that number of "6000" specifically?

A mistype? From what I know, meatReserve should be good to go, and 6000 seems like a bit of overkill since meatReserve should already cover it.

Reply:
6000 is not a misstype, it explains it in the suggestion itself via comment
`//we need 5000 meat for the skill. and want to save an additional 1000 meat above meat reserve since torso awareness is somewhat low priority`

the idea is to make it so that after we buy this we will have at least 1000 remaining on hand.
this is both to delay a bit on its use for torso so that other things can be  bought first. and also because there is an issue with restore code where it won't restore if our meat is under 1000.

and we can not work around this issue with restore code by just adding 1000 to the meatreserve function... since it is also being called in some of the restore code. so someone needs to dig through restore code to figure out that issue. and that code is rather confusing to parse